client/package.json

```diff
@@ -4,6 +4,9 @@
   "private": true,
   "proxy": "http://localhost:3000",
   "dependencies": {
+    "@fortawesome/fontawesome-svg-core": "^1.2.35",
+    "@fortawesome/free-solid-svg-icons": "^5.15.3",
+    "@fortawesome/react-fontawesome": "^0.1.14",
```

Comment:
Please remove all of these dependencies.

Reply:
@ashishnagpal2498 , please check the new PR